### PR TITLE
Update processes.md

### PR DESCRIPTION
**Pull request by HappyBonita**
Return from Mariel&PYM

### CODE REVIEW

File: md/processes.md [hunk 1]
```diff
@@ -54,6 +54,8 @@ To resolve a process:
 
 ## Disable a process
 
+Disable a process can block the creation of new cases, this does not impact existing cases and the use for call activities.
```
Comment (educhastenier) [inline md/processes.md]: Disab**ling** ...

Comment (educhastenier) [reply md/processes.md]: From version 7.4.0 on, a disabled process cannot be started by a call-activity anymore. This was a bug fix.

**Review by educhastenier — CHANGES_REQUESTED** [review]

**Comment by baptistemesta** [comment]
Please follow the commit rules ( commit message and description )
https://bonitasoft.atlassian.net/wiki/display/BS/Contribute+to+Bonita+BPM
https://github.com/bonitasoft/bonita-developer-resources/blob/master/CONTRIBUTING.MD

**Comment by HappyBonita** [comment]
i create a new commit

**Review by educhastenier — APPROVED** [review]

**Pull request merged**